Detections/SecurityAlert/Massdownload_USBFileCopy.yaml

```diff
@@ -0,0 +1,59 @@
+id: 6267ce44-1e9d-471b-9f1e-ae76a6b7aa84
+name: Mass download with new USB file copy activity
```

Comment:
How would this sound as a name "Mass Download & copy to USB device by single user"